index.html

```diff
@@ -877,9 +877,16 @@ <h3>Contact us</h3>
 </section>
     </div>
     <div id="up">
-     <span class="glyphicon glyphicon-chevron-up"></span>
+          <span class="glyphicon glyphicon-chevron-up"></span>
+
     </div>
 
+
+<!--Contact Us-->
+
+<div class="react-card">
```

Comment:
What's the role of this div added?

Reply:
Sorry galat push kar diya I was trying something out,delete karna bhul gaya ghar jaake dekhta hun

Reply:
Cool. Rest Seems OK :+1: